### DISCUSSION TITLE
Help wanted in layout

### DISCUSSION THREAD

**Discussion by TanTai97o** [Sketchware-Pro/Sketchware-Pro#995] [Q&A] [answered]
I cannot add any other layout to the previously added layouts
This error still occurs with the Sketchware version RC-05

https://github.com/Sketchware-Pro/Sketchware-Pro/assets/107351062/2596016b-2e6d-4757-9672-36b7d97a9b2e

**Comment by ilyassesalama** [comment]
So you mean this issue doesn't exist only in the redesigned version of Sketchware Pro?

**Comment by TanTai97o** [comment]
> So you mean this issue doesn't exist only in the redesigned version of Sketchware Pro?

It happens more often with material Redesign versions

**Comment by j-px** [comment]
you need to share that project for us to look into that issue, so please do that in the form of a .swb file (rename it to .zip before uploading)

**Comment by elfilibusterismo** [comment]
If I'm not mistaken your layout has an scrollview? This thing already exists in vanilla sketchware.

**Comment by TanTai97o** [comment]
> you need to share that project for us to look into that issue, so please do that in the form of a .swb file (rename it to .zip before uploading)

[WeistoreProjectDemo.zip](https://github.com/Sketchware-Pro/Sketchware-Pro/files/12779638/WeistoreProjectDemo.zip)

**Comment by elfilibusterismo** [comment]
> that

You can achieved putting a layout there by making your root layout(id = body) match parent(Don't use fixed height)

**Comment by TanTai97o** [comment]
> If I'm not mistaken your layout has an scrollview? This thing already exists in vanilla sketchware.

This error still occurs when there is no scrollview

**Comment by elfilibusterismo** [comment, marked as answer]
> > If I'm not mistaken your layout has an scrollview? This thing already exists in vanilla sketchware.
> 
> This error still occurs when there is no scrollview

Achieving that is a little bit complicated.
See the video...

 https://github.com/Sketchware-Pro/Sketchware-Pro/assets/113604229/190cbb97-62d0-4c2a-8fde-1810aef7d03d

**Comment by TanTai97o** [comment]
> you need to share that project for us to look into that issue, so please do that in the form of a .swb file (rename it to .zip before uploading)



> > > Nếu tôi không nhầm thì bố cục của bạn có chế độ xem cuộn? Thứ này đã tồn tại trong phần mềm phác thảo vanilla.
> > 
> > 
> > Lỗi này vẫn xảy ra khi không có chế độ xem cuộn
> 
> Để đạt được điều đó thì hơi phức tạp một chút. Xem video...
> 
>  Record_2023-10-02-12-55-48_423e58f25c30a02453fa5d8c82fc37f2.mp4

Oh,Thank you very much
I will try it

**Comment by TanTai97o** [comment]
> > > Nếu tôi không nhầm thì bố cục của bạn có chế độ xem cuộn? Thứ này đã tồn tại trong phần mềm phác thảo vanilla.
> > 
> > 
> > Lỗi này vẫn xảy ra khi không có chế độ xem cuộn
> 
> Để đạt được điều đó thì hơi phức tạp một chút. Xem video...
> 
>  Record_2023-10-02-12-55-48_423e58f25c30a02453fa5d8c82fc37f2.mp4

Thank you very much !
